### PR TITLE
Update angular demos to the latest angular version: number 12

### PR DESCRIPTION
**Pull request by carla-at-wiris**
## Description

This PR updates the angular version on angular demos to the latest one, the number 12:

* CKEditor demo: Updated the angular versions to the latest and changes some steps on the demo. Those are specific for our demo configuration, but not to make mathtype work with angular
* Froala demo: Updated the angular versions to the latest and some instructions on the readme file that were outdated.
* Tinymce demo: Updated just the angular versions to the latest.

You'll see that there are no changes on the generic demo. This is because angular V12 updates webpack to V5, which includes some breaking changes that need to be treated carefully on a more specific task. More information on the latest webpack version and how to migrate from V4 to V5 [here](https://webpack.js.org/migrate/5/).

## Steps to reproduce

1. Go to the desired demo folder
2. Run `npm i && npm run start`
3. The demo will open with the latest angular version.

---

[#taskid 13700](https://wiris.kanbanize.com/ctrl_board/2/cards/13700/details/)

### CODE REVIEW

**Review by dcanet-at-wiris — CHANGES_REQUESTED** [review]
Everything looks fine.

But I believe these (2) need to be reviewed:

- I guess we should bump 'patch' version on the package.json demos.

- Also it would be necessary to update the CHANGES.md file.

And I have a question:

- Shouldn't the angular+generic demo be updated, too?

Thanks!

**Review by dcanet-at-wiris — APPROVED** [review]

File: demos/angular/ckeditor5/src/app/app.component.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import * as ClassicEditor from '../ckeditor';
 import * as Generic from 'resources/demos/angular-imports';
 
 // Load wiris plugin version.
-import { version as pluginVersion } from '@wiris/mathtype-ckeditor5/package.json';
+import packageInfo from '@wiris/mathtype-ckeditor5/package.json';
```
[the remark this answers is not in the document]
Comment (carla-at-wiris) [reply demos/angular/ckeditor5/src/app/app.component.ts:9]: The new angular version forces to change the way the parameters from the package.json are obtained. I also checked and we have other files in the project that follow the same methodology.

[the remark this answers is not in the document]
Comment (xripoll-at-wiris) [reply demos/angular/ckeditor5/src/app/app.component.ts:9]: Okay, if it is consistent, it's good

**Review by xripoll-at-wiris — APPROVED** [review]

**Comment by dcanet-at-wiris** [comment]
Ready to merge!

**Pull request merged**